Classes/Widgets/BrokenLinksWidget.php

```diff
@@ -107,4 +122,32 @@ protected function getSearchFields(): array
         }
         return $searchFieldMapping;
     }
+
+    protected function getClause(): string
+    {
+        if ($this->clause) {
+            return $this->clause;
+        }
+
+        $pageClause = $this->getBackendUser()->getPagePermsClause(Permission::PAGE_SHOW);
+
+        $queryBuilder = GeneralUtility::makeInstance(ConnectionPool::class)->getQueryBuilderForTable('tx_linkvalidator_link');
```

Comment:
Why we use here 'tx_linkvalidator_link' ? May be 'pages' ?

Reply:
thanks :+1: